ballerina/Ballerina.toml

```diff
@@ -1,25 +1,25 @@
 [package]
 org = "ballerina"
 name = "io"
-version = "1.6.1"
+version = "1.6.2"
 authors = ["Ballerina"]
 keywords = ["io", "json", "xml", "csv", "file"]
 repository = "https://github.com/ballerina-platform/module-ballerina-io"
 icon = "icon.png"
 license = ["Apache-2.0"]
-distribution = "2201.8.0"
+distribution = "2201.11.0"
 
 [platform.java17]
 graalvmCompatible = true
 
 [[platform.java17.dependency]]
 groupId = "io.ballerina.stdlib"
 artifactId = "io-native"
-path = "../native/build/libs/io-native-1.6.1.jar"
-version = "1.6.1"
+path = "../native/build/libs/io-native-1.6.2-SNAPSHOT.jar"
+version = "1.6.2"
```

Comment:
since we are releasing this with U11, not sure if we need to have minor version bumps instead?

Reply:
Fixed with [cd3bc8bacfcffc4ff18a7f404280d6148e6b8798](https://github.com/ballerina-platform/module-ballerina-io/pull/1300/commits/cd3bc8bacfcffc4ff18a7f404280d6148e6b8798)